pwm/user_data/user_data.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+set -x
+exec > >(tee /var/log/user-data.log|logger -t user-data ) 2>&1
+echo BEGIN
+date '+%Y-%m-%d %H:%M:%S'
+
+# Setup ECS
+echo "ECS_CLUSTER=${ecs_cluster_name}" >> /etc/ecs/ecs.config
+service docker start
+start ecs
+
+# Install tools
+sudo -i
+yum install -y git wget yum-utils jq
+easy_install pip
+PATH=/usr/local/bin:$PATH
+pip install ansible==2.6 virtualenv awscli boto botocore boto3 passlib
+
+# Download user inventory and run ansible
+/usr/bin/curl -o ~/users.yml https://raw.githubusercontent.com/ministryofjustice/hmpps-delius-ansible/master/group_vars/dev.yml
+sed -i '/users_deleted:/,$d' ~/users.yml
+cat << EOF > ~/requirements.yml
+---
+
+- name: bootstrap
+  src: https://github.com/ministryofjustice/hmpps-bootstrap
+  version: centos
+- name: users
+  src: singleplatform-eng.users
+- name: pwm
+  src: https://github.com/ministryofjustice/hmpps-delius-core-pwm-bootstrap
+EOF
+cat << EOF > ~/vars.yml
+---
+
+region: "${region}"
+environment_name: "${environment_name}"
+project_name: "${project_name}"
+ldap_protocol: "${ldap_protocol}"
+ldap_host: "${ldap_host}"
+ldap_port: "${ldap_port}"
+ldap_bind_user: "${ldap_bind_user}"
+user_base: "${user_base}"
+site_url: "${site_url}"
+config_location: "${config_location}"
+
+EOF
+cat << EOF > ~/bootstrap.yml
+---
+
+- hosts: localhost
+  vars_files:
+   - "{{ playbook_dir }}/users.yml"
+   - "{{ playbook_dir }}/vars.yml"
+  roles:
+     - bootstrap
+     - users
+     - pwm
+EOF
+cat << EOF > /etc/sudoers.d/webops
```

Comment:
This is redundant, the users role takes care of this.

Reply:
Removed